tshirts.py

```diff
@@ -9,6 +9,10 @@ def size(cms):
 
 
 assert(size(37) == 'S')
+assert(size(38) == 'S')

```

Comment:
the tests expect the production code to classify size 38 as both S and M. Is that possible? also for 42...